File: programers/javascript/2016년.js
console.log(new Date());

// const date = new Date(2016, 0, 1);
// month는 0~11 까지의 숫자가 입력되어야하고, 0 을 입력하면 1월임을 주의
// date.setFullYear(date.getFullYear() - 1);

// Question
const a = 5;
const b = 24;

function solution(a, b) {
  const answer = [];
  const week = ["sun", "mon", "tue", "wed", "thu", "fri", "sat"];

  const date = new Date(2016, a - 1, b);
  answer.push(week[date.getDay()].toUpperCase());
  return answer[0];
}

// console.log(`기준일자 : ${date}`);

File: programmers/level1/2016년.py
#
#
# 2021/07/01 12:32 오후

import datetime

a, b = 5, 24


def solution(a, b):
    week = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]
    idx = datetime.date(2016, a, b).weekday()
    return week[idx]


print(solution(a, b))

# ClearTime = 2021/07/01 12:54 오후
